syncd/FlexCounterManager.cpp

```diff
@@ -107,9 +107,10 @@ void FlexCounterManager::removeCounter(
 
     fc->removeCounter(vid);
 
-    if (fc->isEmpty())
-    {
-        removeInstance(instanceId);
-    }
+    // TODO: Temporary remove this to avoid flex counter cannot be active
```

Comment:
why this is disabled ? it will affect all counters

Reply:
Yes, it just a temporary fix for MACsec Counter and this BUG has been fixed by this PR: https://github.com/Azure/sonic-sairedis/pull/679 . So I will remove this temporary fix